server/plugin.go

```diff
@@ -410,6 +410,12 @@ func (p *Plugin) OnActivate() error {
 
 	p.apiClient = pluginapi.NewClient(p.API, p.Driver)
 
+	config := p.apiClient.Configuration.GetConfig()
+	license := p.apiClient.System.GetLicense()
+	if !pluginapi.IsE20LicensedOrDevelopment(config, license) {
```

Comment:
Out of scope of this PR but we may want to remane this function check in the plugin api as it was misleading that it was just checking E20 (after looking it checks for enterprise too)